clients/web/static/lib/bootstrap/css

```diff
@@ -1 +0,0 @@
-../../built/
```

Comment:
I changed things to create symlinks to the two moved files during the build process at https://github.com/girder/girder/pull/1161/files#diff-97825ec1d67dac5aa1eb3375f2dd68e4R174 . This way has a few benefits:
* All the compatibility symlinks are in the same place
* Since it's part of the build process, it doesn't clutter the repository with extra directories until build time
* Since individual files are linked, instead of a whole directory, it's more precise and prevents people from abusing the link
* Since it's part of the build process, in theory it might be easier to selectively turn off